routes/index.js

```diff
@@ -314,7 +309,9 @@ module.exports = function (passport) {
           }
         });
       }
-      res.redirect('/userManagement/' + BagdeID);
+      req.flash('success', 'This user is promoted successfully');
```

Comment:
Change this message to "The user has been successfully promoted to Manager status". We want to be pretty verbose with our messages so there is no room for confusion.